src/Forms/Prism.Forms/Services/Dialogs/DialogResult.cs

```diff
@@ -6,19 +6,14 @@
 namespace Prism.Services.Dialogs
 {
     /// <summary>
-    /// An <see cref="IDialogResult"/> that contains <see cref="IDialogParameters"/> from the dialog
-    /// and the <see cref="System.Exception"/> of the dialog.
+    /// Provides returns information about a dialog that has been closed.
```

Comment:
looks like a copy/past issue. Please remove the extra "returns" word.

Reply:
Sorry, I remove this word.
https://github.com/PrismLibrary/Prism/pull/2761/files#diff-0d0a9b1f968cdc60f0420e45398c06795f2b4d473269f729f6a2f977c96ac071R9